ecs/example/main.tf

```diff
@@ -37,6 +37,24 @@ module "cluster" {
   tags = "${var.tags}"
 }
 
+# ------------------------------------------------------------------------------
+# ecs/spotfleet
+# ------------------------------------------------------------------------------
+module "cluster" {
+  source = "../spotfleet"
+
+  prefix              = "${var.prefix}"
+  vpc_id              = "${module.vpc.vpc_id}"
+  subnet_ids          = ["${module.vpc.private_subnet_ids}"]
+  subnet_count        = 3
```

Comment:
`subnet_count` is only passed to `spot-request` from `spotfleet`, but it's not used inside `spot-request`. If the generated spot requests are hardcoded to require 3 subnets (as you said), then we don't need to pass in the variable at all. It will just confuse users and ourselves in the future :p Instead just add a `gotcha` which states that it requires exactly 3 subnets.

Reply:
Not sure how long we will need to wait before we can generate the fleets on demand(years?).  I would prefer to leave the variables in with default values and comments (ie. remove the subnet_count from the example)  as the rest of the code is ready to do this correctly later.